src/modules/70-pipeline/pages/execution/ExecutionPolicyEvaluationsView/utils.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import { defaultTo, forEach, includes, map } from 'lodash-es'
+import { GovernanceMetadata } from 'services/cd-ng'
+import { Evaluation } from 'services/pm'
+import { EvaluationStatus, IacmFormattedData } from './types'
+
+/* istanbul ignore next */
+const defaultEmpty = (value: string | number | undefined): string => defaultTo(value, '') as string
```

Comment:
You can probably simplify here with `unknown`
```suggestion
const defaultEmpty = (value: unknown): string => defaultTo(value, '') as string
```

Or if you're confident that `value` really will be either missing or a string...
```suggestion
const defaultEmpty = (value?: string): string => defaultTo(value, '')
```